### PR TITLE
chore: rename and move pladapter module [DET-5142]

### PR DESCRIPTION
**Pull request by hamidzr**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Review by shiyuann — APPROVED** [review]
Looks good!

File: .circleci/config.yml [hunk 1]
```diff
@@ -235,9 +235,9 @@ commands:
               cat <<parameters.extra-requirements-file>> >> /tmp/cachefile
             fi
 
-      - restore_cache:
```
Comment (shiyuann) [inline .circleci/config.yml:238]: Why do you need to change this?

Comment (hamidzr) [reply .circleci/config.yml]: it was just testing in a temporary commit. should be gone now

**Review by shiyuann — APPROVED** [review]
LGTM!

**Pull request merged**